cmd/mps-control-daemon/mps/manager.go

```diff
@@ -85,6 +85,13 @@ func (m *manager) Daemons() ([]*Daemon, error) {
 			klog.InfoS("Resource is not shared", "resource", "resource", resourceManager.Resource())
 			continue
 		}
+		// Check if MIG devices are included.
+		for _, device := range resourceManager.Devices() {
+			if device.IsMigDevice() {
+				klog.Warning("MPS sharing is not supported for MIG devices; skipping daemon creation")
+				continue
+			}
+		}
```

Comment:
In the happy path we should never reach this point because of the check on applying the `mps.capable=true` label only if MIG is disabled, but good to have this logic here anyway for the transient case of the label still being there from previous runs (or GFD lagging in its application of an updated label).

Reply:
Yes, and a user could in theory force the label too.